### PR TITLE
Add version flag

### PR DESCRIPTION
**Pull request by rawsyntax**

### CODE REVIEW

File: rpenv.go [hunk 1]
```diff
@@ -12,10 +12,19 @@ import (
 	"syscall"
 )
 
+const AppVersion = "2.0.1"
+
 func main() {
-	cmdStatus := 1
+	cmdStatus :=  1
```
Comment (colinrymer) [inline rpenv.go]: Need to remove extra space between `=` and `1`

**Comment by colinrymer** [comment]
Once that nit is resolved, 🚢 it.

**Comment by Teapane** [comment]
LGTM 💃, but I will let Colin chime in.

File: rpenv.go (hunk 1 above)
Comment (Teapane) [inline rpenv.go]: `go fmt` for the win 💃

**Comment by rawsyntax** [comment]
well TIL about `go fmt`

**Pull request merged**